code/datums/mood_events/needs_events.dm

```diff
@@ -91,3 +91,40 @@
 	description = "There's nothing like the feeling of a freshly laundered jumpsuit."
 	mood_change = 2
 	timeout = 10 MINUTES
+
+
+//nutrition
+/datum/mood_event/ate_event
+	timeout = 10 MINUTES
+
+/datum/mood_event/ate_event/no_table
```

Comment:
remove the negative events

Reply:
alternatively, move them to a quirk that grants points. if you do, lower the downside to max -2, -20 is out of control

Reply:
i am not removing the negative events